Update the Czech translation

GeorgeK1ng opened the pull request:
1) 100% Czech translation
Fixed previous translation text - castle building names and all troops. Fixed typos, improved and corrected text on existing 52% translation - touched almost all strings.
Finished remaining 48% of non-translated text. Tested with v1.1.2 public version and original Heroes of Might and Magic II Gold Czech files by CDPROJEKT.

Also tested few maps and game play looks fine. Will see what will be fixed or improved in future. Now is game perfectly playable for Czech users!

GeorgeK1ng commented:
Hi, no one from developers speak Czech. Virtual Keyboard could be saparated, but it's like 10 lines of code only. 
But it makes no sence with whole translation, I used POEdit, everything was using gived rules. I already told this on discord. They should discuss it in internal team. I have spend time to make game play for Czech community. Revised whole text in current translation which was unusable and really broken.
I will polish this 100% in future when needed, then of course in small chunks. I'm not going to make 10 PRs if we cound with 400 lines in each just for review rule. As nobody from developers speak Czech it's up to me. I made whole translation review and fullfilled it to make it 100% and usable to gaming. I have also played various maps and tested it. It works like a charm. File diff looks fine and it's just addition of 100% translation. I hope they can do exception to accept whole translation at once. Otherwise it was waste of time to make this translation.

ihhub commented:
Hi @GeorgeK1ng ,

each pull request should be dedicated to solve one particular thing. This is true for almost every software development project and it is being done to focus only one thing during pull request review and also to keep track of changes in the future. Therefore, please create a separate pull request regarding the virtual keyboard changes.

Regarding the translation part and the size of this pull request. Even despite of a fact that possibly none of developers from the core team speak native Czech, we still review the code as we do for every single translation. This means that the amount of work for checking 2000 lines is gigantic and more code / lines to review, the higher chance of missing any mistakes in the code. Having multiple pull requests is not a problem at all as it is simply a copy-pasted text approach. Knowing line numbers from the file gives precise answer what to copy.

We do not blindly accept any changes in the code without reviewing them. This means that 2 people must manually review the pull request by reading every single line of the changes. You can calculate the amount of time it takes by multiplying (the amount of time to read every single line changed) by (the number of people reviewing it) and (by a factor of 2-4 since it is not native language for us). I think it would give you a rough estimation how much time we would need to review this big text.

Moreover, we can also ask @agiat86 and @joyinko to review the changes but I highly doubt that anyone is willing to read 4000+ lines of text.

GeorgeK1ng commented:
@ihhub I have removed Czech Virtual Keyboard from that branch.

@joyinko @agiat86 Can you please review my whole translation at once? I have touched that file only using POEdit. I also compared it with "empty" translation file when I removed every translation and then compare with Beyond Compare. I don't see any issue in full translation formatting as on POEdit was used to touch it. I hope for positive answer.

joyinko commented on files/lang/cs.po:
```diff
@@ -50,25 +50,22 @@ msgid "INFO"
 
 msgid "QUIT"
-msgstr "KONEC"
+msgstr "UKONČIT"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -80,22 +77,22 @@ msgid "CANCEL"
 
 msgid "ACCEPT"
-msgstr "PŘIJMI"
+msgstr "PŘIJMOUT"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -80,22 +77,22 @@ msgid "CANCEL"
 
 msgid "LEARN"
-msgstr "UČIT"
+msgstr "NAUČIT"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -80,22 +77,22 @@ msgid "CANCEL"
 
 msgid "TRADE"
-msgstr "ZAMĚŇ"
+msgstr "SMĚNIT"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -104,19 +101,19 @@ msgid "NO"
 
 msgid "EXIT"
-msgstr "PRYČ"
+msgstr "ODEJÍT"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -104,19 +101,19 @@ msgid "NO"
 
 msgid "DISMISS"
-msgstr "PROPUSŤ"
+msgstr "PROPUSTIT"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -165,26 +162,26 @@ msgid ""
 
 msgid "CONFIG"
-msgstr "CONFIG"
+msgstr "NASTAVENÍ"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -165,26 +162,26 @@ msgid ""
 msgstr ""
-"DATADISK\n"
-"KAMPAŇ"
+"KAMPAŇ\n"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

GeorgeK1ng replied:
V originálu je to přesně takto a vešlo se to tam.

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -312,16 +310,15 @@ msgstr ""
 
 msgid "RUMORS"
-msgstr ""
+msgstr "POVĚSTI"
```
ZVESTI nebo KLEPY by se hodilo taky?

GeorgeK1ng replied:
Použil jsem pověsti všude, pasuje mi to více do hry.

joyinko commented on files/lang/cs.po:
```diff
@@ -1053,27 +1035,25 @@ msgid "AI"
+msgstr "Zahájit bitvu."
 
 msgid "Exit"
-msgstr "Pryč"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

joyinko commented on files/lang/cs.po:
```diff
@@ -1053,27 +1035,25 @@ msgid "AI"
 
 msgid "Exit"
-msgstr "Pryč"
+msgstr "Odejít"
```
Zde bude text moc dlouhy. Proto jsem texty v prekladu zkracoval, koukal si jak tohle vypada ve hre?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -1728,14 +1699,14 @@ msgid ""
 
 msgid "\"The Dwarfbane!!!!, run for your lives.\""
-msgstr "\"Skřítkobijec!!!!, běžte jako o život.\""
+msgstr "„Trpasličí zhoubaaaa!!! Utíkejte o život!“"
```
zhoubaaaa je takhle napsano zamerne?

GeorgeK1ng replied:
Jo, můžeme klidně přepsat.

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -2027,29 +1992,25 @@ msgid "The %{building} produces %{monster}."
-#, fuzzy
 msgid "Cannot build %{name}. The castle is too far away from an ocean."
-msgstr "%{name} nelze postavit, protože hrad je moc daleko od moře."
+msgstr "%{name} nelze postavit, protože hrad je moc daleko od more."
```
moře - bylo spravne

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3204,17 +3142,16 @@ msgid "Show extra information such as FPS and current time."
 
 msgid "Hotkey: "
-msgstr ""
+msgstr "Klávesová zkratka "
```
Klávesová zkratka:

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3909,910 +3817,843 @@ msgid "Best Hero:"
 
 msgid "Best Hero Stats:"
-msgstr "Statistiky nej. hrdiny:"
+msgstr "Statistiky hrdiny:"
```
Tohle by meli byt statistiky nejepsiho hrdiny. **Best** Hero.

GeorgeK1ng replied:
Na té obrazovce se zobrazují jen statistiky nejlepšíh hrdinů, pokud se nepletu.

GeorgeK1ng commented:
Updated mentioned things. Mostly things to polish during longer gameplays.

joyinko commented:
Hi guys, I commented some rows already yesterday. 
I will need more time to finish it unfortunately.
Anyways, the translation quality and fixes are so far good, very good.

Thank you @GeorgeK1ng

Joyinko


 ** Sun, 13 Oct 2024 04:24:48 -0700 - "ihhub/fheroes2" ***@***.***>, "ihhub/fheroes2" ***@***.***> **

 Hi @GeorgeK1ng ,
 
 each pull request should be dedicated to solve one particular thing. This is true for almost every software development project and it is being done to focus only one thing during pull request review and also to keep track of changes in the future. Therefore, please create a separate pull request regarding the virtual keyboard changes.
 
 Regarding the translation part and the size of this pull request. Even despite of a fact that possibly none of developers from the core team speak native Czech, we still review the code as we do for every single translation. This means that the amount of work for checking 2000 lines is gigantic and more code / lines to review, the higher chance of missing any mistakes in the code. Having multiple pull requests is not a problem at all as it is simply a copy-pasted text approach. Knowing line numbers from the file gives precise answer what to copy.
 
 We do not blindly accept any changes in the code without reviewing them. This means that 2 people must manually review the pull request by reading every single line of the changes. You can calculate the amount of time it takes by multiplying (the amount of time to read every single line changed) by (the number of people reviewing it) and (by a factor of 2-4 since it is not native language for us). I think it would give you a rough estimation how much time we would need to review this big text.
 
 Moreover, we can also ask @agiat86 and @joyinko to review the changes but I highly doubt that anyone is willing to read 4000+ lines of text.
 
 -- 
 Reply to this email directly or view it on GitHub:
 https://github.com/ihhub/fheroes2/pull/9189#issuecomment-2408939205
 You are receiving this because you were mentioned.
 
 Message ID: ***@***.***>

GeorgeK1ng commented:
@joyinko Thanks, I'm sure there will be of course needed a touch. But it will be like quality fixes, to impove game play. I played about 30 hours and fixed what I found. I had no time to test Campaings now. This I can do later on my tablet when 100% translation will be included in Android APK to make it "portable" fo travelling instead of gaming on PC :)

joyinko commented on files/lang/cs.po:
```diff
@@ -2484,67 +2444,55 @@ msgstr "Laboratoř"
 
 msgid "2x Upg. Dwelling 6"
-msgstr ""
+msgstr "Upg. Obydlí 6"
```
"2x Upg. Obydlí 6"

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3453,28 +3385,32 @@ msgid ""
 
 msgid "Click to apply the selected resolution."
-msgstr "Kliknutím se rozliśení změní na vybrané. "
+msgstr "Kliknutím se rozliśení změní na vybrané."
```
"Kliknutím se rozlišení změní na vybrané."

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3522,227 +3458,196 @@ msgid "File to Load:"
-#, fuzzy
 msgid "Select Ocean Object:"
-msgstr "Artefakt"
+msgstr "Vyberte oceánský object:"
```
"Vyberte oceánský objekt:"

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3909,910 +3817,843 @@ msgid "Best Hero:"
-#, fuzzy
 msgid "The map is corrupted."
-msgstr "Soubor je požkozen."
+msgstr "Mapy je poškozena."
```
"Mapa je poškozena."

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3909,910 +3817,843 @@ msgid "Best Hero:"
-msgstr ""
-"Podmínky\n"
-"pro výhru"
+msgstr "Podmínky pro výhru"
```
Nejsou tohle speciální podmínky?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3909,910 +3817,843 @@ msgid "Best Hero:"
-msgstr ""
-"Podmínky\n"
-"pro prohru"
+msgstr "Podmínky pro prohru"
```
Nejsou tohle speciální podmínky?

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3909,910 +3817,843 @@ msgid "Best Hero:"
-#, fuzzy
 msgid "Set alliances:"
-msgstr " aliance"
+msgstr "Nastevení aliancí:"
```
"Nastavení aliancí:"

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3909,910 +3817,843 @@ msgid "Best Hero:"
-#, fuzzy
 msgid "Player Type"
-msgstr "Velikost mapy"
+msgstr "Tyh hráče"
```
"Typ hráče"

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -3909,910 +3817,843 @@ msgid "Best Hero:"
-#, fuzzy
 msgid "Passability"
-msgstr "Osobnost:"
+msgstr "Mřížka"
```
"Průchodnost"

GeorgeK1ng replied:
Opraveno, ale stejně mi přijde, že ta mřížka tam pasuje lépe.

joyinko commented on files/lang/cs.po:
```diff
@@ -4930,537 +4774,508 @@ msgid "The number of days spent on this campaign."
 
 msgid "View High Scores for Campaigns."
-msgstr ""
+msgstr "Zobrazir nejlepší výsledky kampaně."
```
"Zobrazit nejlepší výsledky kampaně."

joyinko commented on files/lang/cs.po:
```diff
@@ -4930,537 +4774,508 @@ msgid "The number of days spent on this campaign."
 
 msgid "hotkey|select extra large map size"
-msgstr ""
+msgstr "Vybra extra velkou mapu"
```
"Vybrat extra velkou mapu"

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -4930,537 +4774,508 @@ msgid "The number of days spent on this campaign."
 
 msgid "hotkey|shipyard"
-msgstr "Loděnice"
+msgstr "Přístap"
```
"Přístav"

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -4930,537 +4774,508 @@ msgid "The number of days spent on this campaign."
 
 msgid "hotkey|join stacks"
-msgstr ""
+msgstr "Sloučit jednotku"
```
"Sloučit jednotky"

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -5565,43 +5380,41 @@ msgstr "Vítej!"
 msgstr ""
-"Vítej ve hře Heroes of Might and Magic II běžící na enginu fheroes2! Před "
-"samotným startem je potřeba nastavit rozlišení."
+"Vítej ve hře Heroes of Might and Magic II běžící na enginu fheroes2!\n"
```
všude je teď vykání, takže **Vítejte**

joyinko commented on files/lang/cs.po:
```diff
@@ -6118,7 +5940,8 @@ msgstr "Zobrazí systémové menu, s volbami pro úpravu chování hry."
 "One or more heroes may still move, are you sure you want to end your turn?"
 msgstr ""
-"Jeden nebo více hrdinů mohou ještě popojet, určitě chceš ukončit svůj tah?"
+"Jeden nebo více hrdinů se ještě může pohybovat, určitě chceš ukončit svůj "
```
"Jeden nebo více hrdinů se ještě může pohybovat, určitě chcete ukončit svůj "

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -6187,14 +6010,14 @@ msgid ""
 "shows the date."
 msgstr ""
-"Zobrazuje informaci o stavu Tvého království nebo hrdinech, dále zobrazuje "
+"Zobrazuje informaci o stavu tvého království nebo hrdinech, dále zobrazuje "
```
vašeho, nebo nic

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -6187,14 +6010,14 @@ msgid ""
 "shows the date. Left click here to cycle through these windows."
 msgstr ""
-"Toto okénko poskytuje informaci o stavu Tvého království, zobrazuje datum. "
+"Toto okénko poskytuje informaci o stavu tvého království, zobrazuje datum. "
```
vašeho, nebo nic

joyinko commented on files/lang/cs.po:
```diff
@@ -6301,195 +6124,195 @@ msgstr "Příjem království za den."
 msgid "For every lighthouse controlled, your ships will move further each day."
 msgstr ""
-"S každým dalším obsazeným majákem budou Tvé lodě moci doplout každý den o "
+"S každým dalším obsazeným majákem budou tvé lodě moci doplout každý den o "
```
vaše

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -6544,7 +6366,7 @@ msgid "To cast spells, you must first buy a spell book for %{gold} gold."
 
 msgid "Unfortunately, you seem to be a little short of cash at the moment."
-msgstr "Bohužel to vypadá, že Ti momentálně schází penízky."
+msgstr "Bohužel to vypadá, že ti momentálně schází zlato."
```
vám

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -6559,16 +6381,16 @@ msgid "two"
 
 msgid "A whirlpool engulfs your ship. Some of your army has fallen overboard."
-msgstr "Vodní vír pohltil Tvou loď. Část posádky spadla přes palubu."
+msgstr "Vodní vír pohltil tvou loď. Část posádky spadla přes palubu."
```
vaší loď

GeorgeK1ng replied:
Opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -7498,319 +7582,375 @@ msgid ""
-#, fuzzy
 msgid "The spell was not found."
-msgstr "Toto kouzlo nebude na nikoho učinkovat!"
+msgstr "Toto kouzlo nebude na nikoho učinkovat."
```
Překlad je Kouzlo nenalezeno, nevím v jakém kontextu toto vyskakuje?

GeorgeK1ng replied:
Taky zatím netuším, opraveno.

GeorgeK1ng commented on files/lang/cs.po:
```diff
@@ -4930,537 +4774,508 @@ msgid "The number of days spent on this campaign."
 
 msgid "Programming Lead"
-msgstr ""
+msgstr "Vedoucí programáto"
```
Opraveno

GeorgeK1ng commented on files/lang/cs.po:
```diff
@@ -5993,109 +5817,107 @@ msgstr ""
-#, fuzzy
 msgid "View a summary of your Kingdom."
-msgstr "Zobraz přehled o svém království."
+msgstr "Zobraz přehled království."
```
Opraveno

GeorgeK1ng commented:
Uploaded cs.po with all mentioned fixes.

joyinko commented on files/lang/cs.po:
```diff
@@ -9693,567 +9818,714 @@ msgid ""
 
 msgid "Endless Purse of Gold"
-msgstr ""
+msgstr "Nekonečná brašna zlata"
```
bezedná?

GeorgeK1ng replied:
Bezedná tobolka zlata, opraveno.

joyinko commented on files/lang/cs.po:
```diff
@@ -10262,208 +10534,265 @@ msgid ""
 
 msgid "Endless Pouch of Sulfur"
-msgstr ""
+msgstr "Nekonečná brašna síry"
```
bezedná?

GeorgeK1ng replied:
Bezedná mošna síry, opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -10262,208 +10534,265 @@ msgid ""
 
 msgid "Endless Vial of Mercury"
-msgstr ""
+msgstr "Nekonečná ampule rtuti"
```
bezedná?

GeorgeK1ng replied:
Bezedná ampule truti, opraveno.

joyinko commented on files/lang/cs.po:
```diff
@@ -10262,208 +10534,265 @@ msgid ""
 
 msgid "Endless Cord of Wood"
-msgstr ""
+msgstr "Nekonečná otep dřeva"
```
bezedná?

GeorgeK1ng replied:
Bezedný sáh dřeva, opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -10262,208 +10534,265 @@ msgid ""
 
 msgid "Endless Cart of Ore"
-msgstr ""
+msgstr "Nekonečný vozík s kamením"
```
bezedný?

GeorgeK1ng replied:
Bezedný vozík kamení, opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -10262,208 +10534,265 @@ msgid ""
 
 msgid "Endless Pouch of Crystal"
-msgstr ""
+msgstr "Nekonečný sáček krystalů"
```
bezedný?

GeorgeK1ng replied:
Bezedná mošna krystalů, opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -10708,21 +11107,28 @@ msgid ""
 
 msgid "Spade of Necromancy"
-msgstr ""
+msgstr "Lopata nekromancie"
```
Rýč, lopata je shovel. Pokud to nebyl záměr...

GeorgeK1ng replied:
Rýč nekromancie, opraveno

joyinko commented on files/lang/cs.po:
```diff
@@ -10793,61 +11199,61 @@ msgid ""
-"stvůru s opět polovičním poškozením a tak dále, dokud není příliš slabý na "
-"to, aby byl škodlivý. Pozor: Kouzlo může zasáhnout i Tvou stranu!"
+"Způsobí zásah vybrané jednotky bleskem elektrické energie ale následně "
+"zasáhne nejbližší stvůru s polovičním poškozením a poté zasáhne DALŠÍ "
```
jednotku/y místo stvůry používej všude, pokud to chceš zaměnit

GeorgeK1ng replied:
Opraveno

joyinko reviewed (commented):
@GeorgeK1ng , já jsem hotov. Prosím mrkni na poznámky, případně uprav pokud uznáš za vhodné. Ďekuji
@ihhub, I am done here, left some comments for GeorgeK1ng. Just minor stuff. I am not playing fheroes at the moment so not sure about text fitting on the tiles. That is for our contirbutor, I guess. I was only focused on translation itself. Please let me know if you need me to approve this pull request? thanks

GeorgeK1ng commented:
@joyinko I'm creating righ now map in original Heroes 2 with all artefacts to make them 1:1 to make it like original translator (Gremlin) did it. Sad is I have no documentation available for artefacts. I will rewrite it in few hours and upload modded file. Every other note was closed and fixed. Great polishment start.

GeorgeK1ng commented on files/lang/cs.po:
```diff
@@ -9693,567 +9818,714 @@ msgid ""
 
 msgid "Endless Bag of Gold"
-msgstr ""
+msgstr "Nekonečná brašna zlata"
```
Bezedná taška zlata, opraveno

GeorgeK1ng commented on files/lang/cs.po:
```diff
@@ -10262,208 +10534,265 @@ msgid ""
 
 msgid "Endless Pouch of Gems"
-msgstr ""
+msgstr "Nekonečný sáček drahokamů"
```
Bezedná mošna drahokamů, opraveno

joyinko reviewed (approved):
Approved, for future please follow the rules and keep ammount of changes at reasonable size.
thx

GeorgeK1ng commented:
@ihhub I hope we can get this translation alive in v1.1.3 if possible.

GeorgeK1ng commented:
@ihhub I have added 2 new lines translated which you merged into my branch tonight. Now it's 100% Czech again.

ihhub commented on files/lang/cs.po:
```diff
@@ -490,9 +488,6 @@ msgid ""
 "combat spells."
 msgstr "Artefakt Sféra Negace znemožňuje v této bitvě použít jakékoliv kouzla."
 
-msgid "You cannot cast spells without a commanding hero."
```
Revert this change back.

GeorgeK1ng replied:
Fixed

ihhub commented on files/lang/cs.po:
```diff
@@ -1771,199 +1742,198 @@ msgid "campaignBonus|Black Pearl"
 
 msgid "campaignBonus|Advanced Archery"
-msgstr "Pokr. Lukostřelba"
+msgstr "Pokročilá Lukostřelba"
```
Can you check how it looks in the Hero's dialog? Does it fit within the skill area?

GeorgeK1ng replied:
I have already fixed campaign skills here, just waited for merge, then PR update with small changes.

ihhub commented on files/lang/cs.po:
```diff
@@ -2785,64 +2728,60 @@ msgid "without artifacts."
 
 msgid "Exit Castle Options"
-msgstr "Pryč z Možnosti Hradu"
+msgstr "Odejít"
```
Should it be a longer text than just Exit?

GeorgeK1ng replied:
I just copied that logic from original game.

GeorgeK1ng replied:
Fixed as requested

ihhub commented on files/lang/cs.po:
```diff
@@ -3252,47 +3187,48 @@ msgid "Scroll Speed"
 msgstr "Vybrat jazyk:"
 
-#, fuzzy
-msgid "Select Language:"
```
Please revert this change back.

ihhub commented on files/lang/cs.po:
```diff
@@ -3252,47 +3187,48 @@ msgid "Scroll Speed"
+msgstr "Nastavit rychlost posouvání okna."
 
 msgid "Select Game Language:"
 msgstr "Vybrat jazyk:"
```
We differentiate game's language and just language which could be a language of a map.

GeorgeK1ng replied:
Fixed

ihhub commented on files/lang/cs.po:
```diff
@@ -3522,227 +3458,196 @@ msgid "File to Load:"
 msgid "Resurrection"
-msgstr "Oživení"
-
-#, fuzzy
```
Revert this change back.

GeorgeK1ng replied:
Reverted, even it's perfectly translatable.

ihhub commented on files/lang/cs.po:
```diff
@@ -3909,910 +3817,843 @@ msgid "Best Hero:"
-"\n"
-"Cesta: "
-
-msgid ""
```
Revert this change back and the one below.

GeorgeK1ng replied:
Fixed

ihhub commented on files/lang/cs.po:
```diff
@@ -3909,910 +3817,843 @@ msgid "Best Hero:"
-msgstr "Událost"
-
-#, fuzzy
-msgid "Click to change the language of the map."
```
Revert this change back.

GeorgeK1ng replied:
Fixed

ihhub commented on files/lang/cs.po:
```diff
@@ -7459,37 +7523,56 @@ msgid_plural ""
 msgid ""
 "You hear a voice from high above in the tower, \"Go away! I can't help you!\""
-msgstr ""
+msgstr "Z věži slyšíte hlas: „Jdi pryč! Nemohu ti pomoci!"
```
I don't think symbol `„` exists in the game resources. Also, there is no closing symbol for the phrase.

GeorgeK1ng replied:
Already fixed them here, just waited for merge to PR small update.

ihhub commented on files/lang/cs.po:
```diff
@@ -7459,37 +7523,56 @@ msgid_plural ""
 "You hear a voice from behind the locked door, \"You don't have enough gold "
 "to pay for my services.\""
 msgstr ""
+"Zpoza zamčených dveří slyšíte hlas: „Nemáte dost zlata na zaplacení mých "
```
I don't think symbol `„` exists in the game resources. Also, there is no closing symbol for the phrase.

GeorgeK1ng replied:
Already fixed them here, just waited for merge to PR small update.

ihhub commented on files/lang/cs.po:
```diff
@@ -7322,135 +7337,184 @@ msgid ""
 "to you for %{count} gold.\""
 msgstr ""
+"Démon na vás skočí a dříve, než stačíte vytáhnout meč, má své drápy na vašem "
+"hrdle. „Tvůj život je můj,“ říká. „Prodám ti ho zpět za %{count} zlata."
```
I don't think symbol `„` and `“` exist in the game resources.

GeorgeK1ng replied:
Already fixed them here, just waited for merge to PR small update.

ihhub commented on files/lang/cs.po:
```diff
@@ -7322,135 +7337,184 @@ msgid ""
 "may fight me, or you may fight my servants. Do you prefer to fight my "
 "servants?\""
 msgstr ""
+"V jeskyni narazíte na mocného a groteskního Démona. „Dnes,“ huhlá, „budete "
```
I don't think symbol `„` and `“` exist in the game resources.

GeorgeK1ng replied:
Already fixed them here, just waited for merge to PR small update.

ihhub commented on files/lang/cs.po:
```diff
@@ -7459,37 +7523,56 @@ msgid_plural ""
 "look to be of poor breeding stock. We have many trained war horses which "
 "would aid your riders greatly. I insist you take them.\""
 msgstr ""
+"Hlavní ošetřovatel vám říká: „To je pěkný kůň, kterého máte. Obávám se, že "
```
I don't think symbol `„` exists in the game resources. Also, there is no closing symbol for the phrase.

GeorgeK1ng replied:
Already fixed them here, just waited for merge to PR small update.

ihhub commented on files/lang/cs.po:
```diff
@@ -7459,37 +7523,56 @@ msgid_plural ""
 "mounted soldiers, their horses look shoddy and weak.\""
 msgstr ""
+"Když se přiblížíte ke stáji, hlavní ošetřovatel se objeví a vede pěkného "
+"válečného koně. „Tento kůň vám pomůže urychlit vaše cesty. Ach, za týden se "
```
I don't think symbol `„` exists in the game resources. Also, there is no closing symbol for the phrase.

GeorgeK1ng replied:
Already fixed them here, just waited for merge to PR small update.

ihhub reviewed (changes requested):
Hi @GeorgeK1ng , I left several comments here. I didn't check the whole translation as it is beyond time to be completed in 1 shot. Most of comments fall into 2 categories:
- remove parts of the translation. which very added recently
- usage of `„` and `“` symbols which I think don't exist in the game

Please take a look at the comments.

GeorgeK1ng commented:
@ihhub I understand, that comparing files on GitHub is real pain. Thats why I never use GitHub files compare. To easily check that my latest cs.po is fine, do it localy please. Download cs.po from your repo as cs_old.po, cs.po from my repo as cs_100%.po. Download Beyond Compare if don't have it. It's trial without any limitations. Drag both files into Beyond Compare and scroll. In few minutes you will see what was changed according to original file. It's easy and fast using this way.

Reason why I have not uploaded my updated cs.po with for example Campaing bonuses was simple, I was waiting for merge, to upload small fixes PR instead of huge one.

What you mean with parts that was added recently? You mean your two lines that were pushed to my translated branch?

GeorgeK1ng reviewed (commented):
Fixed needed things

GeorgeK1ng commented on files/lang/cs.po:
```diff
@@ -508,13 +509,12 @@ msgid ""
 msgstr ""
-"Příkop redukuje obrannou sílu útočících jednotek o -%{count} a zpomaluje o "
-"polovinu jejich perimetr pohybu."
+"Příkop snižuje obrannou sílu útočících jednotek o -%{count} a zpomaluje jejich "
+"pohyb."
```
Hi, this translation is fine, it makes sense in Czech and it means what you are talking about. It's still the same Czech sentence as before my touch, just better written.

GeorgeK1ng reviewed (commented):
Nothing to change

GeorgeK1ng commented:
Can we see this in 1.1.4 please?

GeorgeK1ng commented:
Thanks! Lets see what we need to improve it future! I think there will be just minor changes :)
